README.md

```diff
@@ -101,9 +101,11 @@ There's two optional parameters you can add to this file:
 algolia:
     catch_log_exceptions: true
     index_name_prefix: staging
+    connection_timeout: 2
```

Comment:
Sorry I realized it was in the README.md. For some reasons I was convinced it was in the `service.yml`. My bad!